ReadmeSrc/scripts/fetchData.js

```diff
@@ -0,0 +1,89 @@
+require('dotenv').load()
+const fs = require('fs')
+const axios = require('axios')
+const merge = require('lodash/fp/merge')
+const delay = require('delay')
+
+const data = require('../../webpage/src/data.json')
+
+// https://github.com/settings/tokens
+const access_token = process.env.GITHUB_TOKEN
```

Comment:
@MicheleBertoli 

you can do this

> `GITHUB_TOKEN=123abc yarn build`

Reply:
Oh yes, I was more referring about what we tell people to do in the new README:
> 4. Rename ReadmeSrc/.env_example to ReadmeSrc/.env and set a Github access token to retrieve data from Github.